### ISSUE TITLE
Build crashing on RN 61.1 ../node_modules/react-native-maps/lib/ios/AirGoogleMaps/AIRGoogleMapMarker.m:12:9: 'React/RCTImageLoader.h' file not found

### ISSUE THREAD

**Issue by richardfelkl** [react-native-maps/react-native-maps#3093]
# Bug

When doing Xcode build of my project which has react-native-maps dependency it fails on following error:

`../node_modules/react-native-maps/lib/ios/AirGoogleMaps/AIRGoogleMapMarker.m:12:9: 'React/RCTImageLoader.h' file not found

With RN 60.6 everything is fine.

## Environment info

React native info output:

```bash
System:
    OS: macOS Mojave 10.14.6
    CPU: (8) x64 Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz
    Memory: 33.62 MB / 16.00 GB
    Shell: 5.3 - /bin/zsh
  Binaries:
    Node: 12.9.0 - /usr/local/bin/node
    Yarn: 1.17.3 - /usr/local/bin/yarn
    npm: 6.11.2 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.0, DriverKit 19.0, macOS 10.15, tvOS 13.0, watchOS 6.0
  IDEs:
    Android Studio: 3.5 AI-191.8026.42.35.5791312
    Xcode: 11.0/11A420a - /usr/bin/xcodebuild
  npmPackages:
    react: 16.9.0 => 16.9.0
    react-native: 0.61.1 => 0.61.1
```

Library version: 0.25.0

## Steps To Reproduce

1. Upgrade react to 16.9.0 and react-native to 0.61.1
2. pod install
3. build using Xcode

Describe what you expected to happen:

1. Successful build

**Comment by cormander** [comment]
I had this same issue when I upgraded from RN 0.60.5 to 0.61.1. It's due to the new autolinking functionality not being compatible with the old, manually linked build logic. I ended up doing the `react-native init myproject` to start over with the native stuff, and moved in my source files / fixed the various build config settings.

The lions share of the rework as far as iOS goes is in the `Podfile`, you can see the change to mine with this work here:

https://github.com/OurVoiceUSA/HelloVoter/compare/cf44552ab55163d569fc16b099880de27b7bed04..bbc4b71a1b4d483e0c6b7f901b35313fd76081fe#diff-9a2f2f7a0e84ba7650c8dd93b9801277

In a nutshell, `react-native-maps` and other dependancies are removed and done via autolinking. Most notably, `react-native-google-maps` is the exception, it's not autolinked and needs to stay in there, if you use that.

Hope this helps.

**Comment by NicoCevallos** [comment]
check proposed solution on #3142 (comment)
